bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Trajectory/Orbit/Models/Kepler/COE.cpp

```diff
@@ -52,16 +52,71 @@ inline void                     OpenSpaceToolkitAstrodynamicsPy_Trajectory_Orbit
         .def("get_true_anomaly", &COE::getTrueAnomaly)
         .def("get_mean_anomaly", &COE::getMeanAnomaly)
         .def("get_eccentric_anomaly", &COE::getEccentricAnomaly)
-        .def("get_mean_motion", &COE::getMeanMotion, arg("gravitational_parameter"))
-        .def("get_orbital_period", &COE::getOrbitalPeriod, arg("gravitational_parameter"))
-        .def("get_cartesian_state", &COE::getCartesianState, arg("gravitational_parameter"), arg("frame"))
+
+        .def
```

Comment:
for formatting, I think you can put all of these on a single line

Reply:
This is a new style of formatting that @Derollez encouraged for all python bindings

Reply:
Yeah sorry I updated those earlier to align on some format lucas had started elsewhere. No strong opinion on my side. Everything might not be aligned with the same standard everywhere today, we should address this at some point